### PR TITLE
add lsp support for dhall

### PR DESCRIPTION
**Pull request by kryptt**
Latest commit integrates the dhall layer with lsp support 

also turns the treeview in the scala layer into an option...

### CODE REVIEW

File: layers/+lang/scala/config.el
Comment (smile13241324) [inline layers/+lang/scala/config.el:33]: Default setting was "t" before now it's, nil please make it true by default.

File: init.el
Comment (smile13241324) [inline init.el:18]: Why are you disabling TLS 1.3? Please remove this line

**Review by smile13241324 — CHANGES_REQUESTED** [review]
I have revised your PR and noticed some stuff you should change before this can be merged. Please check my individual comments and mention me when you have questions or you are ready for another review.

File: init.el
Comment (kryptt) [reply init.el:18]: :+1:  
yeah, I was a bit lazy, you often just cherry pick the commits so I didn't bother to remove a local fix for some intermittent issues.. I can remove the commit from the branch

File: layers/+lang/scala/config.el
Comment (kryptt) [reply layers/+lang/scala/config.el:33]: good point

**Review by smile13241324 — APPROVED** [review]
Looks good now, will be merged soon.

**Comment by smile13241324** [comment]
Thank you for contributing to spacemacs 💜, I have cherry picked your commit into develop.

**Pull request merged**
